src/app/components/tile/index.html

```diff
@@ -175,4 +175,56 @@
     </sky-demo-page-code>
   </sky-demo-page-example>
 
+  <sky-demo-page-content sectionHeading="UX guidelines">
+    <p>
+      Tiles are containers to organize varied content. They divide pages into multiple categories and group related content. Use tiles to provide access to a wide range of content where users benefit from the ability to reorder, expand and collapse, and personalize tiles. For more details, see <a routerLink="/design/guidelines/content-containers">content container guidelines</a>.
```

Comment:
Should the link to the content container guidelines be in a "Related information" section at the bottom of the page rather than inline here?

Reply:
That might be in our new content guidelines, I honestly can't remember. I have no strong argument for that link to go in either place.

Reply:
I'll create that Related guidelines sections